bluesky_queueserver/manager/qserver_cli.py

```diff
@@ -1238,11 +1238,20 @@ def prepare_qserver_output(msg):
                 d[k] = "{...}"
     return ppfl(msg)
 
-
 def qserver():
     logging.basicConfig(level=logging.WARNING)
     logging.getLogger("bluesky_queueserver").setLevel("ERROR")
 
+    # Determine the backend type
```

Comment:
This is`qserver` application, it is not using the backend parameter. I would suggest, that for prototyping we just use an environment variable, say `QSERVER_NVM_BACKEND` (there could be a better name) and check it directly in the code. The parameter may be easily added later.